src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,122 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 1 << 4;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int size;
+    private int threshold;
+    private int capacity;
+    private Node<K, V>[] table;
+
+    public MyHashMap() {
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (threshold <= size) {
+            table = resize();
+        }
+        int bucket = getBucket(key);
+        Node<K, V> node = table[bucket];
+        Node<K, V> newNode;
+        if (node == null) {
+            table[bucket] = new Node<>(getHash(key),key,value, null);
+            size++;
+            return;
+        }
+        while (node != null) {
+            if (getHash(key) == node.hash && Objects.equals(node.key, key)) {
```

Comment:
```suggestion
            if (getHash(key) == node.hash && Objects.equals(node.key, key)) {
```
Why do you need this operation getHash(key) == node.hash ?
You do compare two time as far as I understand.